### DISCUSSION TITLE
Moving mill-build.com to mill-build.org

### DISCUSSION THREAD

**Discussion by lihaoyi** [com-lihaoyi/mill#3339] [General]
It's a better fit since Mill is an open source project. Still working on setting up proper redirects etc.

**Comment by lefou** [comment]
Sounds good. You should also reserve a matching github org. Sooner or later, it might become interesting to have it to collect thirdparty plugins and stuff. The Mill users, esp. those on Java projects, not necessarily are also `com-lihaoyi` users.

**Reply by lihaoyi** [reply]
Yep, I've taken https://github.com/mill-build

**Reply by roman-mibex-2** [reply]
https://mill-buid.com does not respont. Only the HTTP endpoint redirects.

AHA. The issue is that:

    http://mill-build.com/ -> redirects to mill-build.org and is working
    https://mill-build.com/ -> Does not answer

My browser defaults to https, so the site looks 'dead'.

**Reply by lihaoyi** [reply]
I've fixed the redirects for mill-build.com so they all redirect to mill-build.org. Namecheap redirects didn't work for https, so I had to set up a new github pages with a bunch of HTML pages to do the redirects https://github.com/com-lihaoyi/mill-build-web-redirect

We don't need to keep it up to date, since it's sole purpose is to keep old links working. So it should just be set-and-forget until such a time as we decide the redirects are no longer necessary
